Splash/Splash/Description/DescriptionViewController.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  DescriptionViewController.swift
+//  Splash
+//
+//  Created by Shreya Vinjamuri on 7/26/22.
+//
+
+import UIKit
+
+class DescriptionViewController: UIViewController {
+    let redView = UIView(frame: CGRect(x: 20, y: 100, width: 140, height: 100))
+    let animation = CABasicAnimation()
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        redView.backgroundColor = .systemRed
+        view.addSubview(redView)
+        // Do any additional setup after loading the view.
+        animateRight();
+        animateDown()
+    }
```

Comment:
Are the animation visible?How about moving your animation from `viewDidLoad` to `viewDidAppear`. Usually you are not able to animate the components of your view until it has finished laying out.

Reply:
Hi Yaxuan, the animation does show up. I tried putting it in viewDidAppear, however I don't want the animation to happen every time you go back to the screen if you've already opened it.